CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -150,6 +150,15 @@ public function upgrade_4_7_beta6($rev) {
    *
    * @param string $rev
    */
+  public function upgrade_4_7_beta9($rev) {
```

Comment:
This and the relevant .tpl file will need to be shifted to 4_7_11 as that is the next version